source/main.c

```diff
@@ -17,29 +19,63 @@ int main(void) {
 	Sleep(100);
 
 	Boot_Log("Hello World!");
+	FILE* fp;
+	char str[16];
+	char* filename = "c:\\input.txt";
+
+	fp = fopen(filename, "r");
+	if (fp == NULL) {
+		printf("Could not open file %s", filename);
+		/*return 1;*/
+	}
+	while (fgets(str, 16, fp) != NULL) {
+		Boot_Log("important : %s", str);
+		
 
+	}
+	fclose(fp);
 	mbedtls_aes_context aes;
 
 	unsigned char key[EXAMPLE_AESKEY_LEN_IN_BYTES];
 	/* TODO create an AES Key ; Just set random values here */
 
 
-	unsigned char iv[AES_IV_LENGTH];
+	unsigned char iv[AES_IV_LENGTH] = "aa";
+	unsigned char iv2[AES_IV_LENGTH] = "aa";
```

Comment:
Initialisation Vector must be the same for both encryption and decryption. 
Once we distribute an encrypted data, we also distribute the IV as well, and the decrypted side uses the IV from the sender. 
So, please remove the iv2 and use the iv for decryption too.